content/en/blog/2025/12/locomo-results-v0.2/index.md

```diff
@@ -0,0 +1,200 @@
+
+Since the original Mem0 evaluation of Locomo benchmark was published, the newer OpenAI gpt-4.1-mini LLM was introduced.  In this paper, we also do a comparison between the original gpt-4o-mini and the newer gpt-4.1-mini when used as the eval-LLM.
+
+
+### Vector DB embedder
+
+When using a memory system, the embedder is the essential element that is used to index the memories in a chat history.  The embedder facilitates the retrieval of saved memories to correctly and factually answer questions.  The choice of embedder can significantly influence the quality of the answers provided by the memory system.  The Mem0 evaluation of Locomo benchmark historically uses the OpenAI text-embedding-3-small embedder.  To compare different products, the embedder will be the same for all products under test.
+
+
+### Judge-LLM
+
+The judge-LLM is the chat LLM that is used to judge if the response from the eval-LLM correctly answers the question in the Locomo benchmark.
+
+The choice of judge-LLM can significantly influence the resulting score.  Different LLMs may give false positives or false negatives when judging the same response.  The Mem0 evaluation of Locomo benchmark historically uses the OpenAI gpt-4o-mini as the judge-LLM.  To compare different products, the judge-LLM will be the same for all products under test.
+
+
+### Reranker
+
+When using a memory system, the reranker is an element that is used to re-evaluate the best matching memories that were retrieved by the embedder.  The reranker will re-sort the retrieved results moving the best results to the top.  The reranker provides a second level of evaluation, providing the best set of saved memories to correctly and factually answer questions.  The choice of reranker can significantly influence the quality of the answers provided by the memory system.  The MemMachine v0.2 uses the AWS cohere.rerank-v3-5:0 as the reranker for the Locomo benchmark.
+
+
+### Question categories
+
+The original Locomo benchmark has 5 categories of questions.  The Mem0 evaluation of Locomo benchmark uses questions from 4 of the 5 categories.  The categories used are as follows:
+
+| Category number | Description | Total Questions |
+| --------------- | ----------- | --------------- |
+| **1** | **Single-Hop:** Questions asking for specific facts directly mentioned in the single session conversation. | 282 |
+| **2** | **Temporal Reasoning:** Questions can be answered through temporal reasoning and capturing time-related data cues within the conversation. | 321 |
+| **3** | **Multi-Hop:** Questions that require synthesizing information from multiple sessions. | 96 |
+| **4** | **Open-Domain:** Questions can be answered by integrating a speaker’s provided information with external knowledge, such as commonsense or world facts. | 841 |
+
+
+### LLM-score
```

Comment:
This is a good summary. Feel free to reference the previous blog where I explain in more detail how this works. https://memmachine.ai/blog/2025/09/memmachine-reaches-new-heights-on-locomo/